docs/ai/conceptual/evaluation-libraries.md

```diff
@@ -33,8 +39,19 @@ The evaluation libraries were built in collaboration with data science researche
 | Fluency      | Evaluates grammatical accuracy, vocabulary range, sentence complexity, and overall readability| <xref:Microsoft.Extensions.AI.Evaluation.Quality.FluencyEvaluator> |
 | Coherence    | Evaluates the logical and orderly presentation of ideas | <xref:Microsoft.Extensions.AI.Evaluation.Quality.CoherenceEvaluator> |
 | Equivalence  | Evaluates the similarity between the generated text and its ground truth with respect to a query | <xref:Microsoft.Extensions.AI.Evaluation.Quality.EquivalenceEvaluator> |
-| Groundedness | Evaluates how well a generated response aligns with the given context | <xref:Microsoft.Extensions.AI.Evaluation.Quality.GroundednessEvaluator><br />`GroundednessProEvaluator` |
-| Protected material | Evaluates response for the presence of protected material | `ProtectedMaterialEvaluator` |
+| Groundedness | Evaluates how well a generated response aligns with the given context | <xref:Microsoft.Extensions.AI.Evaluation.Quality.GroundednessEvaluator> |
+| Relevance (RTC), Truth (RTC), and Completeness (RTC) | Evaluates how relevant, truthful, and complete a response is | <xref:Microsoft.Extensions.AI.Evaluation.Quality.RelevanceTruthAndCompletenessEvaluator>† |
+
+† This evaluator is marked [experimental](../../fundamentals/syslib-diagnostics/experimental-overview.md).
+
+### Safety evaluators
+
+Safety evaluators check for presence of harmful, inappropriate, or unsafe content in a response. They rely on the Azure AI Foundry Evaluation service, which uses a model that's fine tuned to perform evaluations.
+
+| Metric             | Description                                                           | Evaluator type               |
+|--------------------|-----------------------------------------------------------------------|------------------------------|
+| Groundedness       | Evaluates how well a generated response aligns with the given context | `GroundednessProEvaluator`   |
+| Protected material | Evaluates response for the presence of protected material             | `ProtectedMaterialEvaluator` |
 | Ungrounded human attributes | Evaluates a response for the presence of content that indicates ungrounded inference of human attributes | `UngroundedAttributesEvaluator` |
 | Hate content | Evaluates a response for the presence of content that's hateful or unfair | `HateAndUnfairnessEvaluator`† |
 | Self-harm content | Evaluates a response for the presence of content that indicates self harm | `SelfHarmEvaluator`† |
```

Comment:
I think it may be better to remove the content suffix from the name of the metric here and below rows. That would align more with the actual names we use in the code and report.

Reply:
Ah yes that would be a good idea. In that case, I would say use the exact names (and casing) that we use in the code (e.g. `Protected material` -> `Protected Material` etc.).